### PR TITLE
CSS refactor to use tailwind and comply with styleguide a bit more.

### LINKED ISSUE #20: [FE] Make main app views use tailwind
## Background

Make using CSS easier.

## Acceptance Criteria

- [ ] All main app views
 - [ ] Home
 - [ ] etc.

### PR DESCRIPTION
**Pull request by microfauna**
This significantly increases bundle size, but supposedly css-purge cleans it up in prod builds 😄 let's figure  #20 

Adam I'm not sure who else to tag for review so for let's do you and @tarzioo  and whoever else wants to tag along. I'll be throwing a few more commits on here in the next day or so but this is 99% of the work.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,45 +1,5 @@
 <template>
-  <div id="app">
+  <div id="app" class="max-w-screen-sm lg:max-w-screen-lg mx-auto px-4 antialiased text-center">
```
Comment (arecvlohe) [inline src/App.vue]: Nice 😍

File: src/styles/screen.scss [hunk 2]
```diff
@@ -1,16 +1,53 @@
...
+}
+
+.fade-in-down-enter-active,
+.fade-in-down-leave-active {
+  @apply transition-all duration-300 ease-in-out;
+}
+
+.fade-in-down-enter,
+.fade-in-down-leave-to {
+  @apply opacity-0 -translate-y-3 transform;
```
Comment (arecvlohe) [inline src/styles/screen.scss]: So succinct 🤓

File: tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  theme: {
+    fontFamily: {
+      body: ['Inter', 'sans-serif'],
+    }
+  },
+  variants: {},
+  plugins: [],
```
Comment (arecvlohe) [inline tailwind.config.js:8]: This is our config for Natives in Tech:
```
module.exports = {
  purge: ["./components/**/*.tsx", "./pages/**/*.tsx"],
  theme: {
    extend: {},
  },
  variants: {},
  plugins: [require("@tailwindcss/typography")],
};
```
I think `purge` is probably the most important but if you want nice typography then that plugin is nice too.

**Review by arecvlohe — DISMISSED** [review]
LGTM

**Comment by arecvlohe** [comment]
> css-purge

Tailwind does this natively now

> Adam I'm not sure who else to tag for review

Sounds good for now until more want to get involved.

Nice work!

File: src/components/BusinessCard.vue [hunk 4]
```diff
@@ -1,34 +1,48 @@
...
+  <div class="flex flex-row border border-gray-600 mb-6 shadow rounded">
+    <div class="border-r border-gray-600 flex-shrink flex flex-row h-64 w-1/4
+      rounded-l overflow-hidden">
+      <!-- TODO(nsahler): Slider -->
+      <img
+        v-if="business.image_assets.length > 0"
+        :alt="business.name"
+        :src="business.image_assets[0].src"
+        class="w-full h-full max-w-full object-cover self-center"
+      >
```
[the remark this answers is not in the document]
Comment (microfauna) [reply src/components/BusinessCard.vue:13]: -_-

File: tailwind.config.js (hunk 3 above)
Comment (microfauna) [reply tailwind.config.js:8]: Good call, I think I'll add typography at least!

File: src/styles/screen.scss (hunk 2 above)
Comment (microfauna) [reply src/styles/screen.scss]: It should be doing a lil bit more finesse here but I can't figure out why it's failing to transition transforms :D I think it's a vue transition issue.

File: src/components/BusinessCard.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (microfauna) [reply src/components/BusinessCard.vue:13]: ignoring bc alt is generated

**Review by arecvlohe — APPROVED** [review]

**Comment by arecvlohe** [comment]
@nicksahler Looks good!

**Pull request merged**